diag_manager/fms_diag_file_object.F90

```diff
@@ -287,6 +292,16 @@ subroutine add_field_and_yaml_id (this, new_field_id, yaml_id)
   endif
 end subroutine add_field_and_yaml_id
 
+!> \brief Adds a field and yaml ID to the file
+subroutine add_buffer_ids (this, buffer_id)
```

Comment:
The name of this routine is a bit misleading.  It only adds one ID, but `ids` is plural in the routine name.

Reply:
https://github.com/NOAA-GFDL/FMS/pull/1311/commits/21f74ed50859c66e613a01d31c00c1169fcd03c8